react/index.html

```diff
@@ -19,7 +20,7 @@
           platformId: PLATFORM_ID,
           domain: DOMAIN,
           language: LANGUAGE,
-          widgetOptions:{},
+          widgetOptions: {},
         };
       })();
 
```

Comment:
Add opening and closing warning comments around the embed code to indicate that the embed code should not be touched